unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -313,14 +309,9 @@ notifyNumbered o = case o of
           <> IP.makeExample' IP.viewReflog
           <> "to undo this change."
 
-prettyRemoteNamespace ::
-  ( RemoteRepo.ReadRepo,
-    Maybe ShortBranchHash,
-    Path.Path
-  ) ->
-  P.Pretty P.ColorText
+prettyRemoteNamespace :: ReadRemoteNamespace -> P.Pretty P.ColorText
 prettyRemoteNamespace =
-  P.group . P.text . uncurry3 RemoteRepo.printNamespace
+  P.group . P.blue . P.text . uncurry3 RemoteRepo.printNamespace
```

Comment:
Other combinators that pretty-print `ReadRepo` / `WriteRepo` render it in blue, so this seemed consistent.